packages/rocketchat-ui/client/lib/chatMessages.js

```diff
@@ -182,6 +182,12 @@ this.ChatMessages = class ChatMessages {
 
 			const msg = input.value;
 			const msgObject = { _id: Random.id(), rid, msg};
+			const lastMessage = ChatMessage.findOne({rid}, { fields: { ts: 1 }, sort: { ts: -1 }});
+			if (msg[0]+msg[1] === '+:') {
```

Comment:
Use `msg.slice(0, 2)` instead of `msg[0]+msg[1]` and check if the emoji exists first:
```javascript
const reaction = msg.slice(1).trim();
if (RocketChat.emoji.list[reaction]) {
...
```